src/arload.py

```diff
@@ -625,7 +625,8 @@ def load_headers(argflag, spect, datlines):
     return fitsdict
 
 
-def load_frames(slf, fitsdict, ind, det, frametype='<None>', msbias=None, trim=True, transpose=False):
+def load_frames(slf, fitsdict, ind, det, frametype='<None>', msbias=None,
+                trim=True, transpose=False, debug=False):
```

Comment:
Is debug used in this algorithm? I can't seem to find it...
Perhaps instead, it can be pulled from armsgs._debug ?